tests/storage/storage.py

```diff
@@ -219,3 +219,34 @@ def xva_export_import(vm: VM, compression: XVACompression):
         if imported_vm is not None:
             imported_vm.destroy()
         vm.host.ssh(f'rm -f {xva_path}')
+
+ImageFormat = Literal['vhd', 'qcow2']
+
+def vdi_export_import(vm: VM, sr: SR, image_format: ImageFormat):
+    vdi = sr.create_vdi(image_format=image_format)
+    image_path = f'/tmp/{vdi.uuid}.{image_format}'
```

Comment:
I have seen usage of /tmp elsewhere will it make the test re-entrant ? may safer tmp dir used to avoid potential overlap